batch_seqr_loader/find_inputs.py

```diff
@@ -23,8 +23,9 @@ def find_inputs(
     bam_buckets: List[str],
     bam_to_realign_buckets: List[str],
     local_tmp_dir: str,
+    work_bucket: str,
     ped_fpath: Optional[str] = None,
-) -> pd.DataFrame:
+) -> Tuple[pd.DataFrame, str]:
     """
     Find input files (.g.vcf.gz, .bam or .cram) in provided buckets,
```

Comment:
This is good for testing now, but is the plan to talk directly to the metadata server to derive this?

Reply:
Will work on that! Wanted it to be able to run from directly provided inputs as well, to test in isolation.